examples/client/object_location.c

```diff
@@ -56,26 +59,27 @@
 #define RES_M_LATITUDE     0
 #define RES_M_LONGITUDE    1
 #define RES_O_ALTITUDE     2
-#define RES_O_UNCERTAINTY  3
+#define RES_O_RADIUS       3
 #define RES_O_VELOCITY     4
 #define RES_M_TIMESTAMP    5
+#define RES_O_SPEED        6
 
 //-----  3GPP TS 23.032 V11.0.0(2012-09) ---------
 #define HORIZONTAL_VELOCITY                  0  // for Octet-1 upper half(..<<4)
 #define HORIZONTAL_VELOCITY_VERTICAL         1  // set vertical direction bit!
 #define HORIZONTAL_VELOCITY_WITH_UNCERTAINTY 2
 
 #define VELOCITY_OCTETS                      5  // for HORITZOL_VELOCITY_WITH_UNCERTAINTY 
-#define DEG_DECIMAL_PLACES                   6  // configuration: degree decimals implementation
 
 typedef struct
 {
-    char     latitude   [5 + DEG_DECIMAL_PLACES]; //"359.12345" frag=5, 9+1=10! degrees +\0
-    char     longitude  [5 + DEG_DECIMAL_PLACES];
-    char     altitude   [5 + DEG_DECIMAL_PLACES];
-    char     uncertainty[5 + DEG_DECIMAL_PLACES];
+    float    latitude;
+    float    longitude;
+    float    altitude;
+    float    radius;
     uint8_t  velocity   [VELOCITY_OCTETS];        //3GPP notation 1st step: HORITZOL_VELOCITY_WITH_UNCERTAINTY
```

Comment:
Yes, I noticed the HORITZOL typo and throught I'd address that in a separate PR, but I just added another commit fixing it to this one.